web/components/feed/feed-answer-comment-group.tsx

```diff
@@ -54,13 +53,22 @@ export function FeedAnswerCommentGroup(props: {
       answerComments.map((c) => c.id).includes(comment.replyToCommentId)
   )
   const commentsList = answerComments.concat(commentReplies)
-
-  const prob = getDpmOutcomeProbability(contract.totalShares, answer.id)
-  const probPercent = formatPercent(prob)
+  const thisAnswerProbOnMount = useRef<number>()
```

Comment:
One simpler way to get this is using useState:
```
const [thisAnswerProbOnMount] = useState(thisAnswerProb.valueOf())
```

Since useState only initializes the value on first render.

Reply:
Good idear